test/midi-util-test.js

```diff
@@ -0,0 +1,320 @@
+var test = require('tape');
+var Util = require('../lib/jsmidgen.js').Util;
+
+test('Midi letter -> pitches', function(t) {
+	var pitch = Util.midi_letter_pitches;
+
+	t.equal(pitch.a, 21, 'Pitch for a should be 21');
+	t.equal(pitch.b, 23, 'Pitch for b should be 23');
+	t.equal(pitch.c, 12, 'Pitch for c should be 12');
+	t.equal(pitch.d, 14, 'Pitch for d should be 14');
+	t.equal(pitch.e, 16, 'Pitch for e should be 16');
+	t.equal(pitch.f, 17, 'Pitch for f should be 17');
+	t.equal(pitch.g, 19, 'Pitch for g should be 19');
+
+	t.end();
+});
+
+test('Midi pitches -> letter', function(t) {
+	var letter = Util.midi_pitches_letter;
+
+	t.equal(letter[12], 'c', 'Letter for pitch 12 should be c');
+	t.equal(letter[13], 'c#', 'Letter for pitch 12 should be c#');
+	t.equal(letter[14], 'd', 'Letter for pitch 12 should be d');
+	t.equal(letter[15], 'd#', 'Letter for pitch 12 should be d#');
+	t.equal(letter[16], 'e', 'Letter for pitch 12 should be e');
+	t.equal(letter[17], 'f', 'Letter for pitch 12 should be f');
+	t.equal(letter[18], 'f#', 'Letter for pitch 12 should be f#');
+	t.equal(letter[19], 'g', 'Letter for pitch 12 should be g');
+	t.equal(letter[20], 'g#', 'Letter for pitch 12 should be g#');
+	t.equal(letter[21], 'a', 'Letter for pitch 12 should be a');
+	t.equal(letter[22], 'a#', 'Letter for pitch 12 should be a#');
+	t.equal(letter[23], 'b', 'Letter for pitch 12 should be b');
+
+	t.end();
+});
+
+test('Midi flattened notes', function(t) {
+	var flattened = Util.midi_flattened_notes;
+
+	t.equal(flattened['a#'], 'bb', 'a# flattened is bb');
+	t.equal(flattened['c#'], 'db', 'c# flattened is db');
+	t.equal(flattened['d#'], 'eb', 'd# flattened is eb');
+	t.equal(flattened['f#'], 'gb', 'f# flattened is gb');
+	t.equal(flattened['g#'], 'ab', 'g# flattened is ab');
+
+	t.end();
+});
+
```

Comment:
Testing arrays seems suboptimal. I'd prefer to check the results of things that use those arrays (i.e. just the public interface), rather than the internal implementation details.